crime-means-assessment/build.gradle

```diff
@@ -17,6 +18,7 @@ def versions = [
         liquibase                          : "4.20.0",
         crimeCommonsVersion                : "2.2.0",
         mockitoInlineVersion               : "5.2.0",
+        cucumberVersion                    : "7.13.0",
```

Comment:
Should use Cucumber BOM here to ensure consistent versioning for all `cucumber.io` dependencies e.g. `testImplementation(platform("io.cucumber:cucumber-bom:7.14.0"))`